content/en/docs/tasks/tools/install-kubectl.md

```diff
@@ -80,21 +80,24 @@ yum install -y kubectl
 
 ### Install using other package management
 
-If you are on Ubuntu or another Linux distribution that support [snap](https://snapcraft.io/docs/core/install) package manager, kubectl is available as a [snap](https://snapcraft.io/) application. 
-
-If you are on Linux and using [Homebrew](https://docs.brew.sh/Homebrew-on-Linux) package manager, kubectl is available for [installation](https://docs.brew.sh/Homebrew-on-Linux#install).
-
 {{< tabs name="other_kubectl_install" >}}
-{{< tab name="Snap" codelang="bash" >}}
+{{% tab name="Snap" %}}
+If you are on Ubuntu or another Linux distribution that support [snap](https://snapcraft.io/docs/core/install) package manager, kubectl is available as a [snap](https://snapcraft.io/) application.
+
+```
 sudo snap install kubectl --classic
```

Comment:
Aside: we probably don't need `sudo` here.

Reply:
Removed